listenbrainz/spark/spark_reader.py

```diff
@@ -34,17 +35,20 @@ def init_rabbitmq_connection(self):
 
 
     def callback(self, ch, method, properties, body):
-        """ Handle the data received from the queue and works accordingly.
+        """ Handle the data received from the queue and
+            insert into the database accordingly.
         """
         data = ujson.loads(body)
-        user_name = next(iter(data[0]))
-        user = db_user.get_by_mb_id(user_name)
-        artists = data[0][user_name]['artists']['artist_stats']
-        recordings = data[0][user_name]['recordings']
-        releases = data[0][user_name]['releases']
-        artist_count = data[0][user_name]['artists']['artist_count']
+        for key, value in data.items():
```

Comment:
this whole block, including the insert_user_stats and logger call should all be indented in this for loop. Even though we only expect there to be one item here, it doesn't matter that we do it in all in the loop.
Use more specific names in the forloop, e.g. `username, metadata`
Confirm with @paramsingh if we should do early return if the user doesn't exist, of if we want to always do the basic_ack on the channel below.